int32 id
bool active

